testssl.sh

```diff
@@ -3933,11 +3933,11 @@ sclient_connect_successful() {
 determine_tls_extensions() {
      local proto addcmd
      local success
-     local alpn=""
+     local alpnParam=""
      local savedir
      local nrsaved
 
-     "$HAS_ALPN" && alpn="h2-14,h2-15,h2"
+     "$HAS_ALPN" && alpnParam="-nextprotoneg h2-14,h2-15,h2"
```

Comment:
Ah, thx.

Since `man s_client` doesn't mention `-alpn` I guess `npnParam` is the one to go with.